test/duplicates.jl

```diff
@@ -4,7 +4,7 @@ module TestDuplicates
 
     dt = DataTable(a = [1, 2, 3, 3, 4])
     udt = DataTable(a = [1, 2, 3, 4])
-    @test isequal(nonunique(dt), [false, false, false, true, false])
+    @test isequal(.!isunique(dt, 1), [false, false, false, true, false])
```

Comment:
`.!` is 0.6-only syntax. To support 0.5 too, this will need to be `map(!, isunique(dt, 1))`, or the significantly uglier `(!).(isunique(dt, 1))`.